Keas.Mvc/ClientApp/components/Equipment/EquipmentContainer.tsx

```diff
@@ -195,6 +244,37 @@ export default class EquipmentContainer extends React.Component<IProps, IState>
       equipment: updateEquipment
     }); 
   }
+
+  private _filterTags = (filters: string[]) => {
+    this.setState({tagFilters: filters});
+}
+
+  private _checkTagFilters = (equipment: IEquipment, filters: string[]) => {
```

Comment:
@laholstege you can probably write this in a more succinct way with .Every() if I understand the logic here.  Something like `filters.every(f => equipment.tags.indexOf(filter) !== -1)` should do it.  Or even get fancy perhaps and go `filters.every(f => equipment.tags.contains(filter))`